components/invitation_modal/invitation_modal_confirm_step_row.jsx

```diff
@@ -61,6 +80,68 @@ export default class InvitationModalConfirmStepRow extends React.PureComponent {
         }
 
+        if (invitation.fixable && invitation.belongsToTeam) {
+            reason = (
+                <div className='reason-with-fix'>
+                    {reason}
+                    {this.state.error &&
+                        <b className='fix-error'>
+                            <FormattedMessage
+                                id='invite.members.is-already-user-not-in-team.error'
+                                defaultMessage='Unable to add member to channels.'
+                            />
+                        </b>}
+                    {this.state.fixing &&
+                        <a className='fixing'>
+                            <FormattedMessage
+                                id='invite.members.is-already-user-not-in-team.fixing'
+                                defaultMessage='Adding member to channels.'
+                            />
+                        </a>}
+                    {!this.state.fixing && !this.state.error &&
+                        <a
+                            className='fix-link'
+                            onClick={this.addUserToChannels}
+                        >
+                            <FormattedMessage
+                                id='invite.members.is-already-user-not-in-team.add-to-channels'
+                                defaultMessage='Add this member to the channels.'
+                            />
+                        </a>}
+                </div>
+            );
+        } else if (invitation.fixable && !invitation.belongsToTeam) {
+            reason = (
+                <div className='reason-with-fix'>
+                    {reason}
+                    {this.state.error &&
+                        <span className='fix-error'>
+                            <FormattedMessage
+                                id='invite.members.is-already-user-in-team.error'
+                                defaultMessage='Unable to add member to team or channels.'
+                            />
+                        </span>}
+                    {this.state.fixing &&
+                        <a className='fixing'>
+                            <FormattedMessage
+                                id='invite.members.is-already-user-in-team.fixing'
+                                defaultMessage='Adding member to team and channels.'
+                            />
+                        </a>}
+                    {!this.state.fixing && !this.state.error &&
+                        <a
+                            className='fix-link'
+                            onClick={this.addUserToChannels}
+                        >
+                            <FormattedMessage
+                                id='invite.members.is-already-user-in-team.add-to-channels'
+                                defaultMessage='Add this member to the team and channels.'
+                            />
+                        </a>}
+                </div>
+            );
+        }
+
```

Comment:
seems like alot of code repetition here. wouldn't it be cleaner if the id/defaultMessage were decided by 'if' and the rendering part would be the same for both cases?